docs/tutorials/talk-to-kafka.rst

```diff
@@ -0,0 +1,688 @@
+
+4. Select the cloud provider and region that you want to run your service on.
+
+5. Select `business-4` as service plan.
+
+5. Enter ``demo-kafka`` as name for your service.
+
+6. Click **Create Service** under the summary on the right side of the console.
+
+The blinking blue icon besides your service name will indicate that service is being provisioned. Once the icon turns solid green, your Apache Kafka service is up and running.
+
+Making a note of the connection parameters
+------------------------------------------
+
+Let's start by copying the connection information for the newly created Apache Kafka service in order to authenticate and authorize to the Kafka broker. 
+
+.. image:: /images/tutorials/kafka-basics/kafka_service_overview.png
+    :alt: Host, port, username, password and other 
+
+Go to the *Overview* page of your Aiven for Apache Kafka service.
+
+* If you are going to connect with SSL authentication:
+
+  * In the *Connection information* section:
+
+    #. If **Authentication Method** is shown, choose **Client Certificate**
+    #. Next to *Access Key*, click **Download** and save the ``service.key`` file.
+    #. Next to *Access Certificate*, click **Download** and save the ``service.cert`` file.
+    #. Next to *CA Certificate*, click **Download** and save the ``ca.pem`` file.
+
+* If you are going to connect using SASL authentication:
+
+  #. Follow the instructions at `Use SASL Authentication with Apache Kafka® <https://docs.aiven.io/docs/products/kafka/howto/kafka-sasl-auth.html>`_ to enable SASL.
+
+  #. In the *Connection Information* section
+
+     #. Select **SASL** as the **Authentication Method**
+     #. Next to *CA Certificate*, click **Download** and save the ``ca.pem`` file
+     #. Note the *Password* required for the SASL, we'll need it for authentication
+
+* Create the keystore ``client.keystore.p12`` and truststore ``client.truststore.jks`` by following  :doc:`our article on configuring Java SSL to access Kafka <../products/kafka/howto/keystore-truststore>`
+
+.. Warning::
+
+  In the below examples, we just pass the name of the keystore and truststore files, but in actual use, the full path should be used.
+
+You can also use the `Aiven command line tool <https://docs.aiven.io/docs/tools/cli.html>`_ to download the files. See the documentation for `avn service user-creds-download <https://docs.aiven.io/docs/tools/cli/service/user.html#avn-service-user-creds-download>`_
+
+Details on the Aiven for Apache Kafka configuration can be found under the :ref:`kafka-tutorial-reference` section.
+
+.. _tutorial-kaka-python-create-a-topic:
+
+Create a topic
+---------------
+
+A topic in Kafka is a named stream of records that is stored within a Kafka cluster. Let's create a Kafka topic. 
+
+If you have a local Kafka instance running, the command to create the topic might be something like this:
+
+.. code:: bash
+
+    bin/kafka-topics.sh --create --topic demo-topic --bootstrap-server localhost:9092
```

Comment:
We should mention that `localhost:9092` needs to be replaced with the server name